localflavor/bg/fields.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+
+from .validators import egn_validator, eik_validator
+
+
+class EGNField(models.CharField):
```

Comment:
The reason I chose acronym is because if you say to some Bulgarian "Uniform civil number" or "Unique citizenship number" (in English) need to think a lot what this means. If you just say "EGN" everybody (in Bulgaria) will understand. I'm not sure if anyone outside of Bulgaria will use this field. If they really need it probably will know what EGN means.

 EGN in Bulgaria is similar to UMCN (Unique Master Citizen number) in countries that were created after the dissolution of Yugoslavia (Bosnia and Herzegovina, Macedonia, Montenegro, Serbia and Slovenia).
It is similar but the term UMCN is assigned to these countries and the algorithm in Bulgaria is different. For that reason I cant use this name also.

About EIK it is similar to EGN but only for legal personalities.

As I said in my earlier comment I can use the name `BGUniformCivilNumberField` instead of `EGNField` and `BGUniformIdentityCodeField` instead of `EIKField`.